.gitignore

```diff
@@ -114,3 +114,10 @@ docbuild/
 
 # generated perf test scripts
 test/perf/scripts/**
+
+# idea specific stuff
+.idea/**/workspace.xml
```

Comment:
Can we just ignore the entire `.idea` directory?

Reply:
Yeah, sure I will ignore the idea folder completely.